test/rational.jl

```diff
@@ -203,6 +203,21 @@ end
     end
 
     @test Rational(rand_int, 3)/Complex(3, 2) == Complex(Rational(rand_int, 13), -Rational(rand_int*2, 39))
+    @test (true//true) / complex(false, true) === 0//1 - 1//1*im
+    @test (false//true) / complex(false, true) === 0//1 + 0//1*im
+    @test (false//true) / complex(true, false) === 0//1 + 0//1*im
+    @test (false//true) / complex(true, true) === 0//1 + 0//1*im
+    @test (true//true) / complex(true, true) === 1//2 - 1//2*im
+    @test (false//true) / complex(true//true, true//true) === 0//1 + 0//1*im
+    @test (true//true) / complex(true//true, true//true) === 1//2 - 1//2*im
+    @test (false//true) / complex(true//false, false//true) === 0//1 + 0//1*im
+    @test (true//true) / complex(true//true, true//false) === 0//1 + 0//1*im
+    @test_throws DivideError (0//1) / complex(0, 0)
+    @test_throws DivideError (1//1) / complex(0, 0)
```

Comment:
This is wrong. The correct result is positive infinite, just like here:

```julia-repl
julia> (1 // 1) / 0
1//0
```

Reply:
With complex numbers, there are many possible infinities, I'm not sure what infinity is correct.

```julia-repl
julia> 1.0/(0.0im)
NaN + NaN*im
```

Reply:
Ref: https://github.com/JuliaLang/julia/issues/22983 
https://discourse.julialang.org/t/why-inv-complex-0-0-0-0-returns-complex-nan-nan/92694/1

Reply:
You're right, sorry.